src/modbus-rtu.c

```diff
@@ -84,8 +83,7 @@ static const uint8_t table_crc_lo[] = {
     0x99, 0x59, 0x58, 0x98, 0x88, 0x48, 0x49, 0x89, 0x4B, 0x8B,
     0x8A, 0x4A, 0x4E, 0x8E, 0x8F, 0x4F, 0x8D, 0x4D, 0x4C, 0x8C,
     0x44, 0x84, 0x85, 0x45, 0x87, 0x47, 0x46, 0x86, 0x82, 0x42,
-    0x43, 0x83, 0x41, 0x81, 0x80, 0x40
-};
+    0x43, 0x83, 0x41, 0x81, 0x80, 0x40};
```

Comment:
Personally, at this point I would prefer having the closing bracket on the next line.
We could also reformat the whole block having 8 or 16 bytes on each line...